pkg/settings/setting.go

```diff
@@ -167,8 +167,8 @@ var (
 	// Deprecated: On removal use kubeconfig-default-ttl-minutes for all kubeconfigs.
 	KubeconfigTokenTTLMinutes = NewSetting("kubeconfig-token-ttl-minutes", "960") // 16 hours
 
-	// RancherWebhookMinVersion is the minimum version of the webhook that rancher will install.
-	RancherWebhookMinVersion = NewSetting("rancher-webhook-min-version", "")
+	// RancherWebhookVersion is the exact version of the webhook that Rancher will install.
+	RancherWebhookVersion = NewSetting("rancher-webhook-version", "")
```

Comment:
I don't know if we want to replace this setting in a patch release since it will break users. 
We will need to add the new setting and deprecate `RancherWebhookMinVersion`

Reply:
Agreed, though I would say we follow semantic versioning very loosely in our releases.
Should I have two environment variables (and thus two fields, etc.)? This could have problems if both are used and have different logic. But if we just keep the name but have a different meaning and logic, then we could do that.

Reply:
@KevinJoiner, I spoke with Caleb and Michael about this. We decided to keep the old setting but make it non-functional and mark as deprecated. It's to be removed in v2.8.
We'll add a `CATTLE_RANCHER_WEBHOOK_VERSION` environment variable. It should only be set in `package/Dockerfile`. Any code that needs to read it will do so using `os.Getenv`.